src/style.css

```diff
@@ -0,0 +1,135 @@
+html {
+  font-family: Roboto, sans-serif;
+  --color-main-text: #060b37;
+  --color-secondary-text: #616070;
+  --color-accent: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+h2 {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+.product-card {
```

Comment:
your card should have paddings instead of `.product-card__img `and `.product-card__text`